python-libraries/nanover-openmm/src/nanover/openmm/converter.py

```diff
@@ -9,11 +9,11 @@
     # Mypy does not find State in the C module; we ignore the error
     from openmm import State  # type: ignore[attr-defined]
 except (ImportError, ModuleNotFoundError):
-    from simtk.openmm import State
+    from openmm import State
```

Comment:
Here, and just below, we do not need the try/except construct anymore. The code used to say, "Try importing `State` from `openmm`; if you cannot find it, import it from `simtk.openmm`."

This is the same as what you did in `nanoverreporter.py`.